mumuki-laboratory.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "rails", "~> 5.1.6"
 
-  s.add_dependency 'mumuki-domain', '~> 6.5.1'
+  s.add_dependency 'mumuki-domain', '~> 9.9.9' # TODO: Update
```

Comment:
Hey, why do we have a fixed reference to mumuki-domain [in Gemfile](https://github.com/mumuki/mumuki-laboratory/blob/master/Gemfile#L42) and this line here?

Reply:
Oops, the line in the Gemfile should not be there! Nice catch